bimmer_connected/state.py

```diff
@@ -211,6 +229,27 @@ def last_update_reason(self) -> UpdateReason:
         """The the reason for the last state update"""
         return UpdateReason(self._attributes['lastUpdateReason'])
 
+    @property
+    @backend_parameter
+    def condition_based_services(self) -> List['ConditionBasedServiceReport']:
+        """Get staus of the condition based services."""
+        if 'condition_based_services' in self.attributes:
+            cbs_str = self.attributes['condition_based_services']
+        elif 'check_control_messages' in self.attributes:
+            cbs_str = self.attributes['check_control_messages']
+            cbs_str = re.search('condition_based_services: (.*)', cbs_str).group(1)
```

Comment:
No doubt that you are doing the right thing here, but just to be sure;
if `cbs_str` is an empty string then all is good.
Else it will contain the message what is wrong, but I haven't seen an example of that yet ;-)

Reply:
I used these examples to decode the string:
* https://github.com/ChristianKuehnel/bimmer_connected/blob/master/test/responses/unknown_NBT/dynamic.json#L10
* https://github.com/ChristianKuehnel/bimmer_connected/blob/master/test/responses/G31_NBTevo/dynamic.json#L9
* https://github.com/ChristianKuehnel/bimmer_connected/blob/master/test/responses/F87_NBTevo/dynamic.json#L9

If you have more examples, let me know, then I would extend it...